src/aoai-api-simulator/src/aoai_api_simulator/models.py

```diff
@@ -131,6 +131,16 @@ class PatchableConfig(BaseSettings):
     latency: Annotated[LatencyConfig, Field(default=LatencyConfig())]
     allow_undefined_openai_deployments: bool = Field(default=True, alias="ALLOW_UNDEFINED_OPENAI_DEPLOYMENTS")
 
+    # Disable all the no-self-argument violations in this function
+    # pylint: disable=no-self-argument
+    @field_validator("simulator_api_key")
```

Comment:
Can you also update the `Field` default for the `simulator_api_key` to be an empty string? I think that would be less confusing in future if we want to update the default